content/develop/quick-references/release-notes.md

```diff
@@ -19,6 +19,65 @@ pip install --upgrade streamlit
 
 </Tip>
 
+The latest version of Streamlit is 1.39.0.
+
+## **Version 1.39.0**
+
+_Release date: October 1, 2024_
+
+**Highlights**
+
+- 🎤 Introducing [`st.experimental_audio_input`](/develop/api-reference/widgets/st.audio_input) to let users record with their microphones!
+- 📍 [`st.pydeck_chart`](/develop/api-reference/charts/st.pydeck_chart#chart-selections) can return selection events!
+
+**Notable Changes**
+
+- 😃 [`st.button`](/develop/api-reference/widgets/st.button), [`st.download_button`](/develop/api-reference/widgets/st.download_button), [`st.form_submit_button`](/develop/api-reference/execution-flow/st.form_submit_button), [`st.link_button`](/develop/api-reference/widgets/st.link_button), and [`st.popover`](/develop/api-reference/layout/st.popover) each have a new parameter to add an icon.
+- 🏢 [`st.logo`](/develop/api-reference/media/st.logo) has a new parameter to adjust the size of your logo.
+- 🧭 [`st.navigation`](/develop/api-reference/navigation/st.navigation) lets you display an always-expanded or collapsible menu using a new `expanded` parameter.
+- ↕️ You can set `height` and `width` for `st.map` and [`st.pydeck_chart`](/develop/api-reference/charts/st.pydeck_chart#chart-selections).
```

Comment:
nit: st.map isn't hyperlinked. The hyperlink for pydeck chart's height and width shouldn't point to selections